springfield/firefox/templates/firefox/more/best-browser.html

```diff
@@ -74,7 +74,7 @@ <h2>{{ ftl('best-browser-a-browser-that-puts') }}</h2>
         {{ ftl('best-browser-there-are-a-few-ways') }}
       </p>
       <p>
-        {{ ftl('best-browser-the-second-is-not-storing', data=url('privacy.notices.firefox'), privacy=url('privacy.notices.firefox')) }}
+        {{ ftl('best-browser-the-second-is-not-storing', data='https://www.mozilla.org/privacy/firefox/', privacy='https://www.mozilla.org/privacy/firefox/') }}
```

Comment:
Do we want to add UTM params here and below, given we're linking cross-site?

Reply:
At the moment some mozorg links have UTMs and some don't. Do we need UTMs on every link to mozorg or just the ones we want to follow up (i.e. VPN)?

If we want them on every mozorg link, probably best to add a sweep task to check the full site.